frontend_lib/src/container/RowWrappers/ListItemRowWrapper.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import { PAGE } from '../../helper.js'
+import ListItemWrapper from '../../component/Lists/ListItemWrapper/ListItemWrapper.jsx'
+
+const ListItemRowWrapper = (props) => {
```

Comment:
judging by the path of the file, it is a generic file.
But it also has a generic name.
So I could expect other row wrappers in the folder, for example ListUserRowWrapper?
I think it would be better to name it base on its specificity, for example, it has a link so maybe "LinkRowWrapper"?

Reply:
Oh I just realized it's ListItemRowWrapper because it wraps the ListItemWrapper component to be used in a TracimTable row.

So it could be nice to contextualize this component with TracimTable. Either by folder structure or by context like for example, the object TracimTable that would regroup every tracim table components